pages/queue.js

```diff
@@ -26,7 +62,36 @@ const Queue = ({ data }) => {
             active={data[`${queueName}-queue`]["active"]}
             waiting={data[`${queueName}-queue`]["waiting"]}
           />
-          <QueueTable queue_name={queueName} />
+
+          <div className="queue-container">
+            <div
+              className="search-container"
+              style={{
+                marginTop: "2em",
+              }}
+            >
+              <h4>Search Book</h4>
+              <div className="cdx-search-input cdx-search-input--has-end-button">
+                <div
+                  className="cdx-text-input cdx-text-input--has-start-icon"
+                  style={{ width: "100%" }}
+                >
+                  <input
+                    onChange={(e) => onSearch(e)}
+                    className="cdx-text-input__input"
+                    type="search"
+                    placeholder="Search By Job ID , Book Title , Username , or Status"
```

Comment:
This should be "Search by Job ID, Title, Username or Status"

Reply:
fixed